### PR TITLE
Sqlcipher installation for ubuntu update v2

### LINKED ISSUE #3129: Provide the instructions on how to install and use SQLCipher 4 on a system that has an older version installed
## Problem Definition

I want to compile Rotki from source and therefore need a new version of sqlcipher. The install script provided is giving trouble. When I am executing the script `install_deps.sh`, I receive the following error output:

`QLCIPHER_EXISTS: 	libsqlcipher.so.0 (libc6,x86-64) => /lib/x86_64-linux-gnu/libsqlcipher.so.0
Deleted temp working directory /usr/rotki/tmp.9kSyjrlAxp`

I already tried to only run the part of the script that clones the github repository of sqlcipher but that also doesn't update the current version (3.26.0).

### System Description

Operating system: Linux Ubuntu
Rotki version: v.1.18.0

### PR DESCRIPTION
**Pull request by nebolax**
Closes #3129

## Checklist

- Created a new script based on install_deps.sh for manual sqlcipher installation
- The script installs sqlcipher locally and requires user to export a variable in order to use local sqlcipher (I haven't found a good way to do it automatically)
- Added this info to the documentation

### CODE REVIEW

File: sqlcipher_ubuntu.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+git checkout v4.5.0
+./configure \
+--enable-tempstore=yes \
+CFLAGS="-DSQLITE_HAS_CODEC -DSQLITE_ENABLE_FTS3 -DSQLITE_ENABLE_FTS3_PARENTHESIS" \
+LDFLAGS="-lcrypto" \
+prefix="$SQLCIPHER_DIR"
+  make
+  sudo make install
+
+  cd "$DIR" || exit 1
```
Comment (LefterisJP) [inline sqlcipher_ubuntu.sh:54]: There is extra space that is not needed here

```suggestion
make
sudo make install

cd "$DIR" || exit 1
```

**Review by LefterisJP — COMMENTED** [review]
A small comment for the script. @kelsos please review when you can.

File: sqlcipher_ubuntu.sh (hunk 1 above)
Comment (nebolax) [reply sqlcipher_ubuntu.sh]: Oh, I've committed these small changes. Is it okay or is it kinda spam of commits?

**Review by kelsos — APPROVED** [review]
This is exactly what I had in mind thank you @nebolax.

**Pull request merged**